### DISCUSSION TITLE
Icon Size Problem

### DISCUSSION THREAD

**Discussion by di0xazin-e** [elrumo/macOS_Big_Sur_icons_replacements#1364] [Help with the website]
(THIS IS SOLVED,  I AM NOT ABLE TO DELETE THIS DISCUSSION BUT I HAVE FOUND A SOLUTION. THANK YOU.)

Greetings all,

I was wanting to upload an icon today and for some reason while doing that, it says for me to check the size of the icon. It told me to make sure that it is 1024px x 1024px and I checked with the project that I was doing it on and it was the said size. 

Not sure if the website is just acting weird on my end or not but any help or advice for this will be sincerely appreciated.

<img width="424" alt="Screen Shot 2022-03-05 at 5 57 36 PM" src="https://user-images.githubusercontent.com/84487353/156905961-f7c462b5-9e2b-4113-934c-44f5bb07135d.png">
<img width="365" alt="Screen Shot 2022-03-05 at 5 57 49 PM" src="https://user-images.githubusercontent.com/84487353/156905964-2c5a2373-db3b-448b-be40-ad805e4b0119.png">

**Comment by SkinnyPig2** [comment]
I'm having the same issue. How did you get it to work?

**Reply by di0xazin-e** [reply]
Oh wow! Didn't expect a message on this thread. Basically when I found this out I couldn't do anything. A day later of when I made this post, I clicked the option "I am the author of this icon" then it started working for me.

**Reply by SkinnyPig2** [reply]
oh huh. Thanks!
